### DISCUSSION TITLE
dynamic tooltip

### DISCUSSION THREAD

**Discussion by iishika-s** [microsoft/vscode-discussions#728] [Extension Development QnA] [answered]
Can we pass a dynamic title for an icon in package.json file? if yes, then how

![image](https://github.com/microsoft/vscode-discussions/assets/93257779/ed655393-b0c8-43a5-8939-8b3e40d8ae2a)

I want to show the no of incoming changes in the tool tip

**Comment by alefragnani** [comment, marked as answer]
Hi @iishika-s ,

AFAIK this is not supported today, as command's `title`/`shortTitle` are constant declared upfront in `package.json`, and doesn't support variables/interpolation at runtime.

On the other hand, you could try the `FileDecoration` API (https://code.visualstudio.com/api/references/vscode-api#FileDecorationProvider), and use its runtime approach to display dynamic contents as its tooltip

Hope this helps
